qualification/report/generate_pdf.py

```diff
@@ -876,17 +894,21 @@ def main():
     parser.add_argument("input", help="Output of pytest --report-log=...")
     parser.add_argument("pdf", help="PDF file to write")
     parser.add_argument("-c", "--commit-id", action="store_true", help="Output commit ID of katgpucbf image")
+    parser.add_argument("--generate-procedure-doc", action="store_false", dest="make_report", default=True)
     parser.add_argument(
-        "--report-doc-id",
+        "--doc-id",
         help="Document number to write to the qualification test report",
         type=str,
-        default="E1200-0000-005",
+        default=DEFAULT_DOC_ID,
     )
     args = parser.parse_args()
+    if args.make_report is False and args.doc_id == DEFAULT_DOC_ID:
```

Comment:
`not foo` is the usual way of testing whether a boolean is false, except in cases where you want to be sure it is exactly the value False and not some falsey non-boolean.

Reply:
I remember thinking about it, and I think my logic was, I wasn't expecting it to be anything other than `False`.

But I've handled the argument validation in a different way now so this line goes away.